extension/src/popup/views/GrantAccess/index.tsx

```diff
@@ -48,6 +48,9 @@ export const GrantAccess = () => {
     window.close();
   };
 
+  const isMalicious =
+    data && "is_malicious" in data ? data.is_malicious : false;
```

Comment:
was getting a type error with the new type as "is_malicious" doesn't exist when Blockaid returns a miss